frontend/src/views/v-authorship.vue

```diff
@@ -97,11 +97,14 @@
       .file(v-bind:key="file.path")
         .title
           span.path(v-on:click="toggleFileActiveProperty(file)")
-            .tooltip
-              font-awesome-icon(icon="caret-down", fixed-width, v-show="file.active")
-              span.tooltip-text(v-show="file.active") Click to hide file details
-              font-awesome-icon(icon="caret-right", fixed-width, v-show="!file.active")
-              span.tooltip-text(v-show="!file.active") Click to show file details
+            span(v-show="file.active")
+              .tooltip
```

Comment:
How about `.tooltip(v-show="file.active")`

Reply:
Yes I think that is better.